### DISCUSSION TITLE
Renovate not finding package config file

### DISCUSSION THREAD

**Discussion by shenke-adsk** [mend/renovate-ce-ee#467] [Q&A]
We have a repository on which Renovate says "Repository has no package files - skipping
   name: renovate". We do have a file under the path "[RS.Core.Model]($ROOT/tree/master/RS.Core.Model)/packages.config". Coincidently, the repository is also named RS.Core.Model. Is that a bug?

**Comment by rarkins** [comment]
It's almost definitely not a bug. Please create a reproduction repository and then raise an "Ask a Question" Discussion here: https://github.com/renovatebot/renovate/discussions
